content/logs/faq/log-collection-troubleshooting-guide.md

```diff
@@ -113,6 +138,10 @@ config_providers:
     polling: true
 ```
 
+### Journald
+
+When using Journald in a containerised environment, make sure to follow the instruction of the [journald integration][8] as there is a specific file to mount to the agent.
```

Comment:
```suggestion
When using Journald in a containerized environment, make sure to follow the instructions in the [journald integration][8] as there is a specific file used to mount to the Agent.
```

Our English documentation is written in `en_US`, so as much as it pains me, _ize_ is preferred. :(